backend/models/postModel.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require('mongoose')
+
+const postSchema = new mongoose.Schema(
+  {
+    userID: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'User',
+      required: [true, 'User ID missing']
+    },
+    text: {
+      type: String,
+      trim: true
+    },
+    image: {
+      type: Array
```

Comment:
Can you explain your reasoning to make image an array type? Were you thinking it would be an array of strings in case a single post can have multiple images?

Reply:
Yes, I made image an array type. A possible way to store our images could be to upload the images to Cloudinary or maybe Amazon s3 bucket and then upload the URL to the database.